emails/tests/views_tests.py

```diff
@@ -1073,14 +1082,20 @@ def setUp(self):
             },
         }
         self.complaint_body = {"Message": json.dumps(complaint)}
+        ses_client_patcher = patch(
+            "emails.apps.EmailsConfig.ses_client",
+            spec_set=["send_raw_email"],
+        )
+        self.mock_ses_client = ses_client_patcher.start()
+        self.addCleanup(ses_client_patcher.stop)
 
     def test_notification_type_complaint(self):
```

Comment:
Hmm, but I like that this test tests the no-flag behavior to make sure that all the existing functionality is the same. What do you think? I'm tempted to just mark this resolved and not worry about it.

Reply:
If adding the `override_flag` I recommend also testing that the mask is not disabled as well.

Reply:
The default behavior will change based on the flag being defaulted to enabled or disabled right? If that's the case I think adding the override here to make it explicit would be better. This would also help when the flag is removed to update this test.